test/unit/app/services/service_listeners/publishing_api_html_attachments_test.rb

```diff
@@ -486,10 +486,16 @@ class Republish < PublishingApiHtmlAttachmentsTest
           attachment.content_id,
           publication.search_link,
           "en",
-          false,
+          true,
         )
         call(publication)
       end
+
+      test "for a publication that has been unpublished does not publish the attachment in order to unpublish again" do
+        publication = create(:unpublished_publication_in_error_no_redirect)
+        PublishingApiWorker.any_instance.expects(:perform).never
+        call(publication)
+      end
```

Comment:
I'm personally not a huge fan of tests that test something doesn't happen, or at least not beyond an initial TDD cycle. Theoretically the set of things we could test don't happen is infinite so I always feel a bit dubious about it.

Reply:
In a documentation sense that we explicitly want to call out that publishing "should not" happen, then I think it should be tested for things that we definitely do not want to happen otherwise bad things happen. To prevent in the future if people do add that functionality back in, they can see the test and thing "hmm do i really want to do this"